lib/cisco_node_utils/bgp.rb

```diff
@@ -327,8 +327,8 @@ def enforce_first_as
       config_get('bgp', 'enforce_first_as', @get_args)
     end
 
-    def enforce_first_as=(enable)
-      @set_args[:state] = (enable ? '' : 'no')
+    def enforce_first_as=(action)
+      @set_args[:state] = (action ? '' : 'no')
```

Comment:
Not a functional change, just 'enable' to 'action' change as XR has the 'disable' keyword so you'd be 'enabling disable'. Which hurts my :bulb: